examples/offline_inference/eagle.py

```diff
@@ -118,8 +118,8 @@ def main():
             acceptance_counts[step] += count
 
     print("-" * 50)
-    print(f"mean acceptance length: \
-        {sum(acceptance_counts) / acceptance_counts[0]:.2f}")
+    print(f"mean acceptance length (including bonus tokens): \
```

Comment:
I thought the bonus token was already accounted for in `acceptance_counts[0]`? Is that not the case? If so, why does `acceptance_counts` have length `num_spec_tokens + 1`?